src/main/java/com/ays/backend/user/model/BaseEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.ays.backend.user.model;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
```

Comment:
please remove unused import.

Reply:
I just removed unnecessary imports from the class